code/Pagetypes/FreshdeskPage.php

```diff
@@ -0,0 +1,262 @@
+<?php
+
+
+class FreshdeskPage extends Page
+{
+}
+
+class FreshdeskPage_Controller extends Page_Controller
+{
+    /**
+     * Status mapping for Freshdesk tickets.
+     *
+     * @var array
+     * @config
+     */
+    private static $freshdeskStatus = [
+        2 => 'open',
+        3 => 'pending',
+        4 => 'resolved',
+        5 => 'closed',
+    ];
+
+    /**
+     * Priority mapping for Freshdesk tickets.
+     *
+     * @var array
+     * @config
+     */
+    private static $freshdeskPriority = [
+        1 => 'low',
+        2 => 'medium',
+        3 => 'high',
+        4 => 'urgent',
+    ];
+
+    /**
+     * @var array
+     */
+    private static $_tickets = [];
+
+    /*
+    * Returns Freshdesk tickets for a user can be filtered by priority or status
+    *
+    * @param Array $filter
+    * @return PaginatedList
+    *
+    */
+    public function getFreshdeskTickets()
```

Comment:
Remove "Freshdesk" from name. Class already hints at the kind of tickets we might get.